index.css

```diff
@@ -284,22 +284,78 @@ background-color: red;
 /* end of section two */
 
 
+/*Start of contributor page styling */
+
+.contributors-body {
+  margin: 1vw;
+}
+
+.contributors-main-section {
+display: grid;
+grid-template-columns: auto auto auto;
+grid-template-rows: auto;
+gap: 2vw;
+}
+
+.contributors-page .hero-section {
+  margin: 10px;
+  font-size: 2rem;
+  text-align: center;
+}
+
+.contributor-card {
+  border: 2px solid var(--medium-gray);
+  border-radius: 8px;
+  text-align: center;
+  
+}
+
+.contributor-avatar {

```

Comment:
Did you play around with the height property on these? I notice we have a variety or image sizes being rendered. It's fine for now!

Reply:
I did with the .img-container, but will open an issue about this. Would love other people to look over add to/or fix some of my code.